sources/core/Xenko.Core.Mathematics/Color.cs

```diff
@@ -525,7 +525,7 @@ public static void Modulate(ref Color left, ref Color right, out Color result)
         /// <returns>The modulated color.</returns>
         public static Color Modulate(Color left, Color right)
         {
-            return new Color(left.R * right.R, left.G * right.G, left.B * right.B, left.A * right.A);
+            return new Color((byte)(left.R * right.R), (byte)(left.G * right.G), (byte)(left.B * right.B), (byte)(left.A * right.A));
```

Comment:
Shouldn't there be a division by 255 here? (i.e. max intensity red would be R = 255 x R = 255 and should result in R = 255)